app/components/cities-nearby/template.hbs

```diff
@@ -0,0 +1,5 @@
+{{#each cities as |city|}}
+  <a href="/map/{{city.inseeCode}}" class="city-nearby">
```

Comment:
i didnt manage to use the LinkTo here, because of the missing parameter.
how to pass parameters ?

i tried: 

```
<LinkTo @route="map.city" @inseeCode={{city.inseeCode}} class="city-nearby">
  {{city.inseeCode}}
</LinkTo>
```